M2/Macaulay2/e/BasicPoly.hpp

```diff
@@ -8,13 +8,17 @@
 #include <iosfwd>
 #include <string>
 #include <unordered_map>
-  
+#include <gmpxx.h>
+
 class BasicPoly
 {
 public:
-  std::vector<int> mCoefficients;
+  std::vector<mpz_class> mCoefficients;
```

Comment:
The function `bytesUsed` will need to change.  We don't really need to do that before this PR is accepted, but it should be changed.